diff -Nur xcdroast-0.98alpha10.orig/init.c xcdroast-0.98alpha10.chng/init.c
--- xcdroast-0.98alpha10.orig/init.c	Tue Apr 23 20:55:12 2002
+++ xcdroast-0.98alpha10.chng/init.c	Mon Jun  3 16:50:52 2002
@@ -84,7 +84,12 @@
 	setupdata.cddb_host = g_strdup("freedb.freedb.org");
 	setupdata.cddb_port = 888;
 
-	g_snprintf(tmp,MAXLINE,"%s/%s", CONFIGDIR, LOGFILE);
+	if (getenv("CONFIG_DIR"))
+		g_snprintf(tmp, MAXLINE, "~/%s/%s/%s", 
+			getenv("CONFIG_DIR"), CONFIGDIR, LOGFILE);	
+	else 
+		g_snprintf(tmp, MAXLINE, "~/.%s/%s", CONFIGDIR, LOGFILE);
+
 	setupdata.logfile = g_strdup(tmp);
 	setupdata.loglevel = 0;
 
@@ -719,7 +724,13 @@
 	ignorescanbus = 0;
 	noversioncheck = 0;
 	alt_dev_string = NULL;
-	strncpy(configdir, CONFIGDIR, MAXLINE);
+	
+	if (getenv("CONFIG_DIR"))
+		g_snprintf(configdir, MAXLINE, "~/%s/%s", 
+			getenv("CONFIG_DIR"), CONFIGDIR);	
+	else 
+		g_snprintf(tmp, MAXLINE, "~/.%s", CONFIGDIR);
+
 	strncpy(rootconfig, ROOTCONFIG, MAXLINE);
 
 #ifdef PRE_LIBDIR 
diff -Nur xcdroast-0.98alpha10.orig/xcdroast.h xcdroast-0.98alpha10.chng/xcdroast.h
--- xcdroast-0.98alpha10.orig/xcdroast.h	Tue Apr 23 20:55:27 2002
+++ xcdroast-0.98alpha10.chng/xcdroast.h	Mon Jun  3 16:36:53 2002
@@ -118,7 +118,7 @@
 /* default paths/configfile */
 #define PREFIX "/usr/local"
 #define LIBDIR "lib/xcdroast-0.98"
-#define CONFIGDIR "~/.xcdroast"
+#define CONFIGDIR "xcdroast"
 #define CONFFILE "xcdr098.cfg"
 #define LOGFILE "xcdr098.log"
 #define ISOOPTFILE "isoopt.cfg"
